src/core/utils/checks/charge_neutrality.hpp

```diff
@@ -0,0 +1,33 @@
+#include <cmath>
+
+#include "PartCfg.hpp"
+
+namespace Utils {
+
+inline void check_charge_neutrality(PartCfg &partCfg) {
+  double total_charge = 0.0;
+  double min_charge = 0.0;
+  for (auto const &particle : partCfg) {
```

Comment:
Reimplement this with `std::accumulate`, `std::min` and `std::max`. Also probably there should be no magic number.

Reply:
Why? Which magic number?

Reply:
`1e-10`

Reply:
that magic number was already there before

Reply:
Isn't that sentence already longer than putting `numeric_limits` instead of the number?